web/src/components/nomination/JobInput.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <v-row dense>
+    <v-col
+      cols="12"
+      md="6"
+      xs="12"
+    >
+      <v-text-field
+        ref="title"
+        label="Job Title"
+        outlined
+        :value="value.title"
+        @input="updateJob('title', $event)"
+      />
+    </v-col>
+    <v-col
+      cols="12"
+      md="6"
+      xs="12"
+    >
+      <v-text-field
+        ref="company"
+        :value="value.company"
+        label="Company"
+        outlined
+        @input="updateJob('company', $event)"
+      />
+    </v-col>
+  </v-row>
+</template>
+
+<script>
+export default {
+  props: {
+    value: {
+      type: Object,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      jobFields: ['title', 'company'],
+    };
+  },
+  methods: {
+    // Build the Job object by populating all the nameFields and latest values
+    updateJob(updatedField, updatedValue) {
```

Comment:
There's an opportunity to DRY things up here, but I would like to revisit it later.

Reply:
Hmmm not only on this method, but could all text inputs be a generic `TextInput` component with some configurable attributes (like field name, placeholder, etc)?

Reply:
Happy to revisit it later, I keep thinking if there is a way, but it might be a fair amount of work. Unfortunately vue is quite verbose, and I am still learning the v2 gridding system.